sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -4,13 +4,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Core" />
     <PackageReference Include="Azure.Identity" />
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="NUnit" />
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
-    <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'net47'"/>
+    <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'net47'" />
```

Comment:
This doesn't ship in client profiles of .NET Framework, not to mention is a pretty beefy dependency. Is there anyway we can avoid this, even if it means duplicating some method?

Reply:
>Starting with the .NET Framework 4.5, the Client Profile has been discontinued and only the full redistributable package is available. Optimizations provided by the .NET Framework 4.5, such as smaller download size and faster deployment, have eliminated the need for a separate deployment package. The single redistributable streamlines the installation process and simplifies your app's deployment options.

I'm not sure it's worth trying to avoid. It seems to be in everyone's GAC

Reply:
What's it for, exactly? It wasn't obvious from the rest of the code. To your point, maybe we ignore it for now and - depending on what you're using it for - we can fix it later if not exposed or too difficult.

Reply:
I'm not adding it, VS just reformatted the line.

Reply:
We use it for `NameValueCollection queryParams = HttpUtility.ParseQueryString(uri.Query);`